perun-base/src/main/java/cz/metacentrum/perun/core/api/MembersOrderColumn.java

```diff
@@ -18,7 +18,39 @@ public enum MembersOrderColumn {
 	                 "users.first_name " + getLangSql(query) + query.getOrder().getSqlValue()
 	),
 
-	ID("", "", query -> "members.id " + query.getOrder().getSqlValue());
+	ID("", "", query -> "members.id " + query.getOrder().getSqlValue()),
+	STATUS("","", query -> "members.status " + query.getOrder().getSqlValue()),
+	GROUP_STATUS("", "", query -> "source_group_status " + query.getOrder().getSqlValue()),
+
+	// 1. user preferred mail, 2. member mail
+	EMAIL(
+			", uav.attr_value, mav.attr_value ",
+			" left join " +
+			   	"(select attr_value, member_id, attr_id from member_attr_values) as mav " +
```

Comment:
the aliases 'mav' and 'uav' are used later on in the WHERE string, could this be an issue (or at the very least be confusing) ?

Reply:
yeah, didn't realize they are used later on. renamed.